rlImGui/rlImGui.cpp

```diff
@@ -84,10 +111,15 @@ static void rlImGuiNewFrame()
     if ((io.ConfigFlags & ImGuiConfigFlags_NoMouseCursorChange) == 0)
     {
         ImGuiMouseCursor imgui_cursor = ImGui::GetMouseCursor();
-        if (io.MouseDrawCursor || imgui_cursor == ImGuiMouseCursor_None)
-            HideCursor();
-        else
-            ShowCursor();
+		if (imgui_cursor != lastMouseCursor || io.MouseDrawCursor) {
+			lastMouseCursor = imgui_cursor;
+			if (io.MouseDrawCursor || imgui_cursor == ImGuiMouseCursor_None)
+				HideCursor();
```

Comment:
if the else has brackets then the if should have brackets too (so they both match, even if the if is one line)